ClinGen/clincoded
# Connects to #1476. Allow Mammalian Phenotype Ontology MP IDs as additional option in Experimental/Model Systems

Steps to test:

1. Login to VCI/GCI, Create New Gene-Disease Record (GCI) => Enter PMID to access right-side tab "Evidence for PMID"
2. Curate Experimental Data Information => Experiment type: Model Systems
3. Phenotype(s) observed in model system: MP or HP IDs should pass form validation when submitting. Only MP/HP IDs are allowed to be submitted through form validation.
4. View/Score Experimental Data => Phenotype(s) observed in model system: MP ID has no valid link-out, should be plain text. HP ID should have a corresponding link-out.
